tests/views/playingScreen-test.js

```diff
@@ -38,17 +43,62 @@ describe('PlayingScreen', function () {
     expect(clicked).toBe(true);
   });
 
-  it('creates a PlayingScreen and checks touchEnd', function () {
+  it('creates a PlayingScreen and checks mouseDown, mouseUp with video360', function() {
+    var isTouched = false;
+    var isStartHideControlBarTimer = false;
+    var mockController = {
+      state: {
+        isMobile: false,
+        accessibilityControlsEnabled: false,
```

Comment:
Do we have a defined behavior when accessibility controls are enabled? We should add some tests to test the defined behavior if we do have it defined.

If we do not have a defined behavior, what happens when accessibility controls are enabled? Do we have any conflicts with keyboard controls? For example, arrow keys might be used to navigate between elements when they are enabled.